stdlib/http/server.pyi

```diff
@@ -52,25 +52,15 @@ class BaseHTTPRequestHandler(socketserver.StreamRequestHandler):
 class SimpleHTTPRequestHandler(BaseHTTPRequestHandler):
     extensions_map: dict[str, str]
     if sys.version_info >= (3, 12):
-        def __init__(
-            self,
-            request: socketserver._RequestType,
-            client_address: socketserver._AddressType,
-            server: socketserver.BaseServer,
-            *,
-            directory: str | None = ...,
-            index_pages: Sequence[str] | None = ...,
-        ) -> None: ...
-    else:
-        def __init__(
-            self,
-            request: socketserver._RequestType,
-            client_address: socketserver._AddressType,
-            server: socketserver.BaseServer,
-            *,
-            directory: str | None = ...,
-        ) -> None: ...
-
+        index_pages: tuple[str, ...]
```

Comment:
I see you refer to `index_pages` as a "class variable" in the news entry for the CPython change. If the attribute is only meant to be set on the class itself, not on instances, we can represent it by using the special `ClassVar` annotation from the `typing` module. 

```suggestion
        index_pages: ClassVar[tuple[str, ...]]
```